android-appcoins-billing/src/main/java/com/appcoins/sdk/billing/mappers/GamificationMapper.java

```diff
@@ -1,33 +1,36 @@
-package com.appcoins.sdk.billing.payasguest;
+package com.appcoins.sdk.billing.mappers;
 
+import com.appcoins.sdk.billing.models.GamificationModel;
 import com.appcoins.sdk.billing.service.RequestResponse;
 import org.json.JSONArray;
 import org.json.JSONException;
 import org.json.JSONObject;
 
 import static com.appcoins.sdk.billing.utils.ServiceUtils.isSuccess;
 
-class GamificationMapper {
-  GamificationMapper() {
+public class GamificationMapper {
+
+  public GamificationMapper() {
   }
 
-  int mapToMaxBonus(RequestResponse requestResponse) {
+  public GamificationModel mapToMaxBonus(RequestResponse requestResponse) {
     JSONObject jsonObject;
-    int maxBonus = -1;
+    GamificationModel gamificationModel = new GamificationModel();
     int code = requestResponse.getResponseCode();
     String response = requestResponse.getResponse();
-
     if (isSuccess(code) && response != null) {
       try {
         jsonObject = new JSONObject(response);
         JSONArray jsonArray = jsonObject.getJSONArray("result");
         JSONObject level = jsonArray.getJSONObject(jsonArray.length() - 1);
         double maxBonusEntity = level.getInt("bonus");
```

Comment:
you use get int into a double and after that you cast double to int. Redundant cast?